### PR TITLE
feat: add sysdig terraform scripts

### PR DESCRIPTION
**Pull request by junminahn**
- add sysdig Terraform scripts

### CODE REVIEW

File: sysdig-teams/values-c6af30.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+licensePlate: c6af30
```
Comment (thegentlemanphysicist) [inline sysdig-teams/values-c6af30.yaml:1]: Assuming that adding a person here will add that person to sysdig. The github action to cause the change is then automatically triggered by the merge?

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
-# sso-sysdig
\ No newline at end of file
+# sso-sysdig
+
```
Comment (thegentlemanphysicist) [inline README.md:2]: Thank you for the documentation

**Review by thegentlemanphysicist — APPROVED** [review]
Looks good thank you for the added documentation

File: sysdig-teams/values-c6af30.yaml (hunk 1 above)
Comment (junmin-bcgov) [reply sysdig-teams/values-c6af30.yaml:1]: yes, so incase we will need to add/remove members, then will be done via PR & CD pipiline after merge.

**Pull request merged**